widget/embed_example.html

```diff
@@ -8,22 +11,24 @@
 
     <div id="embedded-talks"></div>
     <div id="embedded-table"></div>
+    <div id="embedded-calendar" style="max-width:800px"></div>
         
     <script>
     var params = 
-        {   from: "today",             //Start date range - dd/mm/yy. Required
+        {   from: "01/01/01",             //Start date range - dd/mm/yy. Required
```

Comment:
If it can't be 'today' please see #13 - could we have a better solution
than the beginning of time :-)

On 04/02/2015 13:59, Martin Filliau wrote:
> In widget/embed_example.html
> <https://github.com/ox-it/talks.ox-js-widget/pull/15#discussion_r24084129>:
> 
>>          
>>      <script>
>>      var params = 
>> -        {   from: "today",             //Start date range - dd/mm/yy. Required
>> +        {   from: "01/01/01",             //Start date range - dd/mm/yy. Required
> 
> Why can't it be |today|?
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ox-it/talks.ox-js-widget/pull/15/files#r24084129>.
>